libdnf/repo/Repo.cpp

```diff
@@ -469,6 +469,7 @@ std::unique_ptr<LrHandle> Repo::Impl::lrHandleInitBase()
         dlist.push_back(item.c_str());
     }
     dlist.push_back(NULL);
+    handleSetOpt(h.get(), LRO_PRESERVETIME, 1L);
```

Comment:
It preserve timestamp for all downloaded files. But there is a problem with primary file. It is used for detection when repo was last synced. The computation of repo age must be changed or simply overwrite timestemp of primary file after download.

Reply:
Right. I changed the code so that this feature is not enabled by default, but only on demand.